src/test/operators/export_binary_test.cpp

```diff
@@ -500,7 +500,7 @@ TEST_P(OperatorsExportBinaryMultiEncodingTest, AllTypesNullValues) {
   column_definitions.emplace_back("d", DataType::String, true);
   column_definitions.emplace_back("e", DataType::Double, true);
 
-  auto table = std::make_shared<Table>(column_definitions, TableType::Data, Chunk::MAX_SIZE);
+  auto table = std::make_shared<Table>(column_definitions, TableType::Data, 5);
```

Comment:
Is this necessary to avoid running into the `mvcc->max_begin_cid != MvccData::MAX_COMMIT_ID` assert?

Reply:
No, this is necessary so that we do not try to allocate a couple hundred of GBs for a chunk with the maximum size